lib/talk_helpers.rb

```diff
@@ -6,3 +6,11 @@ def past_and_next_talks items
     i.attributes[:kind] == :talk and i.attributes[:date] <= limit
   end.sort_by { |i| i.attributes[:date] }
 end
+
+def upcoming_talks items
+  limit = Date.today
```

Comment:
So you propose that I call the link "upcoming" rather than "schedule"? If yes, please provide a patch to this PR!